### PR TITLE
Fix goals to be more clear

### LINKED ISSUE #1786: Fix part of README.md to be easier to read and follow
## Summary
The documentation should be clear and easy to read.

## Actual style
https://user-images.githubusercontent.com/45157667/180976891-e6e7dc97-b990-41df-8b8f-305429240549.png

## Expected style
Goals should be listed in a more clear format such as a list of bullet points instead of being packed together in one long paragraph.

### PR DESCRIPTION
**Pull request by KaevonD**
As I am just changing one part of the README.md and not uploading any new files or updating files to the latest version, none of the steps apply.

I made to follow the documentation style with my change. Goals under the vision section changed from a paragraph to a bullet point list to be easier to read and more clear.

Fixes #1786

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,15 @@ docker run --name devdocs -d -p 9292:9292 thibaut/devdocs
 
 DevDocs aims to make reading and searching reference documentation fast, easy and enjoyable.
 
-The app's main goals are to: keep load times as short as possible; improve the quality, speed, and order of search results; maximize the use of caching and other performance optimizations; maintain a clean and readable user interface; be fully functional offline; support full keyboard navigation; reduce “context switch” by using a consistent typography and design across all documentations; reduce clutter by focusing on a specific category of content (API/reference) and indexing only the minimum useful to most developers.
+The app's main goals are to: 
+
+* Keep load times as short as possible
+* Improve the quality, speed, and order of search results
+* Maximize the use of caching and other performance optimizations
+* Maintain a clean and readable user interface; be fully functional offline
```
Comment (ojeytonwilliams) [inline README.md:61]:
```suggestion
* Maintain a clean and readable user interface 
* Be fully functional offline
```
To be consistent with the rest.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks, @KaevonD, this does make things easier to parse. Could you add the last bullet point?

**Comment by KaevonD** [comment]
Ah sorry I missed that last one. Added @ojeytonwilliams !

**Review by ojeytonwilliams — APPROVED** [review]
No problem, @KaevonD. Thanks for sorting that out. This LGTM, now :+1:

**Pull request merged**